app/src/main/res/values/strings.xml

```diff
@@ -250,7 +250,7 @@
     <string name="sort">Sort</string>
     <string name="showMoreInfo">Show info</string>
     <string name="hideMoreInfo">Hide info</string>
-    <string name="swipeToSwitchImages">Swipe or long press to switch images</string>
+    <string name="swipeToSwitchImages">Swipe to switch images\nLong press an image to open in gallery app</string>
```

Comment:
> The Toast message of two lines isn't being displayed correctly on my Galaxy S20FE with regular font size, is there another way of letting the user know about this?

Ah, that's a shame. Thanks for testing to make sure the code works well! Maybe then just:
```suggestion
    <string name="swipeToSwitchImages">Swipe to switch image, hold to open image in gallery app</string>
```